content/forgero-compat/src/main/resources/data/forgero/packs/naturalist/material/glow_goop.json

```diff
@@ -0,0 +1,73 @@
+{
+  "type": "GOO",
```

Comment:
Side note, it seems like it does not show you the stats because it has a custom shift modifier.

Reply:
![image](https://github.com/user-attachments/assets/1920bd93-8ecf-4347-810d-d5a3159bf87b)

Reply:
yea nothing I can do about that atm